.travis.yml

```diff
@@ -1,17 +1,26 @@
 sudo: false
+cache: bundler
 language: ruby
 
 rvm:
-  - 2.0
-  - 2.1
-  - 2.2
-  - jruby
-  - rbx-2
+  - 2.0.0-p648
+  - 2.1.8
+  - 2.2.4
+  - 2.3.0
```

Comment:
This version expansion has been a bit unstable on Travis recently (https://github.com/travis-ci/travis-ci/issues/5581). I usually use the full version number on other projects just to be sure of what is going on.